src/python_testing/TC_AVANALYTestBase.py

```
@@ -0,0 +1,41 @@
+    SPEC_MAX_COUNT_SUPPORTEDAMBIENTCONTEXTS = 50
+
+    has_feature_lclcondetect = False
+    has_feature_remcondetect = False
+    has_feature_perzonedetect = False
+
+    async def read_avanaly_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.AvAnalysis
+        return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
+
+    async def check_avanaly_attribute(self, attribute, expected_value, endpoint):
+        value = await self.read_avanal_attribute_expect_success(endpoint=endpoint, attribute=attribute)
```

Pylint (4.0.7)

[error] Instance of 'AVANALYTestBase' has no 'read_avanal_attribute_expect_success' member; maybe 'read_avanaly_attribute_expect_success'?

(E1101)